### ISSUE TITLE
Add support for Android / iOS targets

### ISSUE THREAD

**Issue by bitspittle** [varabyte/truthish#5]
Technically shouldn't be challenging but I need to spend time figuring out how to set up the CI to build them.

**Comment by tscholze** [comment]
@bitspittle if you try to build iOS targets you need a Mac with Xcode (as you already know). These CI machines are expensive in context of "minute factor points". 

![Bildschirm­foto 2023-03-04 um 19 58 20](https://user-images.githubusercontent.com/5583469/222923944-52d5c383-a99a-4105-9c98-a27cd3ae5e1e.png)

**Comment by bitspittle** [comment]
Thanks for the heads up! I was not aware of the minute multiplier for mac. Ouch! I'll be careful running tests on mac runners!

I checked check my billing page and it's weird -- it says I used 0 minutes so far this month, but I was doing a bunch of churn the last few days. Maybe it just takes a few days to update...

I am already using Mac machines to build Mac Intel and Mac M1 targets. Might as well add iOS targets eventually too :) I don't plan to build these things too often. But I'll keep an eye on my usage!!
